src/Utils/Shadows/index.ts

```diff
@@ -0,0 +1,28 @@
+const ShadowOneDown = '0 2px 4px 0 rgba(0, 0, 0, 0.15)';
+const ShadowTwoDown = '0 4px 8px 0 rgba(0, 0, 0, 0.15)';
+const ShadowThreeDown = '0 6px 12px 0 rgba(0, 0, 0, 0.10)';
+const ShadowFiveDown = '0 12px 24px 0 rgba(0, 0, 0, 0.10)';
+const ShadowFiveUp = '0 -6px 12px 0 rgba(0, 0, 0, 0.10)';
+
+export const Shadows = {
+  1: {
+    down: ShadowOneDown,
+  },
+  2: {
+    down: ShadowTwoDown,
+  },
+  3: {
+    down: ShadowThreeDown,
+  },
+  4: {
+    down: ShadowFiveDown,
```

Comment:
Is it strange that the index is `4` but the var names used here say `Five`?

Reply:
Oops typo. 😅

Reply:
Update the interface to 
```
Shadow.up1
Shadow.down1
Shadow.up2
Shadow.down2
// ...
```
I think using direction is more scaleable if we have left or right in the future.
And even the designers have defined one up shadow, I think we can still add `up1` to `up3` based on the same rule by ourselves first in case it confuses the other devs as this situation https://github.com/glints-dev/glints-aries/pull/555#discussion_r546531565.